introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-fine-tuning-flan-t5.ipynb

```diff
@@ -0,0 +1,596 @@
+    "    initial_instance_count=1,\n",
+    "    instance_type=inference_instance_type,\n",
+    "    predictor_cls=Predictor,\n",
+    "    endpoint_name=pre_trained_name,\n",
+    "    volume_size=30,\n",
+    ")\n",
+    "print(f\"{newline}Deployed an endpoint {pre_trained_name}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "07cab2be-f433-41c3-bee5-e8e0d61a7a91",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from sagemaker.model import Model\n",
+    "from sagemaker.predictor import Predictor\n",
+    "from sagemaker.utils import name_from_base\n",
+    "\n",
+    "fine_tuned_name = name_from_base(f\"jumpstart-demo-fine-tuned-{model_id}\")\n",
+    "fine_tuned_model_uri = f\"{output_location}{training_job_name}/output/model.tar.gz\"\n",
+    "\n",
+    "# Create the SageMaker model instance of the fine-tuned model\n",
+    "fine_tuned_model = Model(\n",
+    "    image_uri=deploy_image_uri,\n",
+    "    model_data=fine_tuned_model_uri,\n",
+    "    role=aws_role,\n",
+    "    predictor_cls=Predictor,\n",
+    "    name=fine_tuned_name,\n",
+    ")\n",
+    "\n",
+    "print(f\"{bold}image URI:{unbold}{newline} {deploy_image_uri}\")\n",
+    "print(f\"{bold}model URI:{unbold}{newline} {fine_tuned_model_uri}\")\n",
+    "print(\"Deploying an endpoint ...\")\n",
+    "\n",
+    "# Deploy the fine-tuned model.\n",
+    "fine_tuned_predictor = fine_tuned_model.deploy(\n",
+    "    initial_instance_count=1,\n",
+    "    instance_type=inference_instance_type,\n",
+    "    predictor_cls=Predictor,\n",
+    "    endpoint_name=fine_tuned_name,\n",
+    "    volume_size=30,\n",
+    ")\n",
+    "print(f\"{newline}Deployed an endpoint {fine_tuned_name}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58114831-c08e-4542-bc5e-852826785356",
+   "metadata": {},
+   "source": [
+    "### 4. Running inference queries"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5ebf33c-dc6f-41a9-b137-1e2ffaa8de2e",
+   "metadata": {},
+   "source": [
+    "As the name suggests, a Text2Text model such as FLAN T5 receives a piece of text as input, and generates text as output. The input text will contain the description of the task. In this demo, our task is to generate questions given a piece of text. The questions must be relevant to the text, but the text should contain no answer. Such a task could arise when automating gathering additional information, or identifying gaps in technical documentation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2598072e-fc52-40c0-8343-1a74e35cfa3a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "prompt = \"Ask a question which is related to the following text, but cannot be answered based on the text. Text: {context}\"\n",
+    "\n",
+    "# Sources: Wikipedia, AWS Documentation\n",
+    "test_paragraphs = [\n",
```

Comment:
Optional: we can make `test_paragraphs` a list of dictionary with `context` field, and make it similar to template formatting.

Reply:
I feel that such a way can be more confusing